Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -27,18 +27,18 @@ BUCKET_FULL_TARGET_PATH="$GCS_BUILD_BUCKET/$BUILD_BUCKET_PATH"
 
 # ====== BUCKET CONFIGURATION  ======
 if [[ -z "$1" ]]; then
-  GCS_MARKET_BUCKET='marketplace-dist'
+  GCS_MARKET_BUCKET=$GCS_PRODUCTION_BUCKET
```

Comment:
Do we have the `GCS_PRODUCTION_BUCKET` env var ordinarily or is it something you just added? If you just added it to the GitLab flow, do the changes in this file work in the CircleCI flow?